phpBB/styles/prosilver/theme/common.css

```diff
@@ -1269,6 +1269,10 @@ ul.linklist:after,
 	padding: 4px 6px;
 }
 
+.badge.hidden {
+	display: none;
+}
+
```

Comment:
Oops... And it would block the ability to customize... Unless another !important is thrown in, not so nice... Maybe use a different class name?